src/logging/build-key-to-redact.ts

```diff
@@ -0,0 +1,10 @@
+const prefix = `["`;
+const suffix = `"]`;
+const joinSeparator = `${suffix}${prefix}`;
+
+export const buildKeyToRedact = (parts: string[]): string => {
+  if (parts.length === 0) {
```

Comment:
should we use:
```
if (!parts.length) {...}
```
this to keep  it consistent with other similar checks

Reply:
Good point, done!